### ISSUE TITLE
Clicking about hangs app

### ISSUE THREAD

**Issue by oempire** [hunkydoryrepair/MonitorKeeper#6]
When i click about in the app it hangs also show window is it meant to be blank?

**Comment by hunkydoryrepair** [comment]
I don't think it hangs. It pops up a very small dialog window that you must close. of course...that dialog window might be on a different monitor.

EDIT: Show Window is really for debug mode. It was accidentally left on the menu for release. The window will be empty in the release build.

**Comment by oempire** [comment]
thanks will check that out - perhaps i didn't see the popup.

Would love to see it a little more integrated - maybe a start with windows (as admin etc)
